site/.gitignore

```diff
@@ -1,6 +1,5 @@
 .DS_Store
 node_modules
-/dist
```

Comment:
Por que o `dist` foi removido do gitignore?

Reply:
Pois o `dist/` é o diretório de build do Vue. tem que deixar ele pra levar pra produção.

Reply:
Então eu não entendi o escopo desse _pull request_, pois ele resolve um monte de coisa.

- Corrige problemas de lint;
- Adiciona um módulo para ler/escrever no JSON;
- Centraliza todos os comandos de contagem;
- Insere o Vue como dependência (embora não esteja no package.json).

Issue ou pull request sem escopo definido não é muito legal para validar/revisar o código.
Não seria melhor tratar a parte do Vue em um pull request específico?

Reply:
> Então eu não entendi o escopo desse _pull request_, pois ele resolve um monte de coisa.
> 
>     * Corrige problemas de lint;
> 
>     * Adiciona um módulo para ler/escrever no JSON;
> 
>     * Centraliza todos os comandos de contagem;
> 
>     * Insere o Vue como dependência (embora não esteja no package.json).
> 
> 
> Issue ou pull request sem escopo definido não é muito legal para validar/revisar o código.
> Não seria melhor tratar a parte do Vue em um pull request específico?

Essa parte já foi tratada inclusive. no ultimo [Pull Request que foi feito por mim inclusive])(https://github.com/levxyca/pandadomalbot/pull/43). mas suponho que este fork deste Pull Request deve ter sido feito antes daquele que eu arrumava exatamente esta questão. Mas se for necessário, o dono deste Pull Request poderia retirar essa mudança mesmo.

Reply:
Não faltou colocar o `vue` nas dependências? 🤔

Reply:
> Não faltou colocar o `vue` nas dependências? thinking

Em qual `package.json` ? Acredito que no `package.json` do bot não faz sentido colocar o Vue como dependência pois são em lugares diferentes. mas na branch do [franciscoemanuel](https://github.com/franciscoemanuel/pandadomalbot/tree/feature/combo-nos-contadores) no arquivo `site/package.json` o Vue e vários outros pacotes são dependências:
```
{
  "name": "site",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "lint": "vue-cli-service lint"
  },
  "dependencies": {
    "core-js": "^3.6.5",
    "vue": "^3.0.0"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "~4.5.0",
    "@vue/cli-plugin-eslint": "~4.5.0",
    "@vue/cli-service": "~4.5.0",
    "@vue/compiler-sfc": "^3.0.0",
    "babel-eslint": "^10.1.0",
    "eslint": "^6.7.2",
    "eslint-plugin-vue": "^7.0.0-0"
  },
  "eslintConfig": {
    "root": true,
    "env": {
      "node": true
    },
    "extends": [
      "plugin:vue/vue3-essential",
      "eslint:recommended"
    ],
    "parserOptions": {
      "parser": "babel-eslint"
    },
    "rules": {}
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not dead"
  ]
}
```

Reply:
Tendeu.

Reply:
@rn4n na verdade eu fiz um pull da main, mas aparentemente a branch de testes está desatualizada, e por isso ocorrem essas diferenças.